### DISCUSSION TITLE
Caching of API results?

### DISCUSSION THREAD

**Discussion by khalavak** [intelowlproject/IntelOwl#1933] [General]
Hello!
I am wondering if Intelowl caches the results returned by the configured APIs so that subsequent lookups use the cached results before going out to the APIs to fetch latest results?

**Comment by mlodic** [comment]
Hello! Thanks for your interest in the project! :)

IntelOwl does not cache the results from the APIs you query _but_ it does cache Jobs Results.
Practically this means that if, for instance, you have analyzed an IP address and, 2 hours later, you try to analyze it again, by default you'll get the data that you have already extracted during the previous analysis (without any new API request)

The default behavior is that the analysis would not be repeated if that has been already done during the last 24 hours.
However, this behavior can be easily changed by the user in the GUI in the "Scan" page in the "Advanced settings" section:

![image](https://github.com/intelowlproject/IntelOwl/assets/30625432/644da3f0-d557-49e9-820e-e7259e886005)

For instance, if you choose "Force new analysis" you would disable this caching mechanism.

**Comment by khalavak** [comment]
Thats great 
Thanks a lot!
